src/main/java/hex/genmodel/easy/EasyPredictModelWrapper.java

```diff
@@ -0,0 +1,291 @@
+package hex.genmodel.easy;
```

Comment:
For reference, I tried this, without success:

````
--- a/pom.xml
+++ b/pom.xml
@@ -39,7 +39,8 @@
                <dependency>
                        <groupId>ai.h2o</groupId>
                        <artifactId>h2o-genmodel</artifactId>
-                       <version>3.0.1.4</version>
+                       <version>3.6.0.8</version>
+                        <scope>provided</scope>
                </dependency>
                <dependency>
                        <groupId>org.reflections</groupId>
@@ -100,6 +101,15 @@
                        </plugin>
                        <plugin>
                                <artifactId>maven-failsafe-plugin</artifactId>
+                                <version>2.19.1</version>
+                                <configuration>
+                                    <classpathDependencyExcludes>
+                                        <classpathDependencyExcludes>ai.h2o:h2o-genmodel</classpathDependencyExcludes>
+                                    </classpathDependencyExcludes>
+                                    <additionalClasspathElements>
+                                        <additionalClasspathElement>src/test/resouces/h2o-genmodel-3.0.1.4.jar</additionalClasspathElement>
+                                    </additionalClasspathElements>
+                                </configuration> 
                                <executions>
                                        <execution>
                                                <id>integration-test</id>
````
...with the listed jar copied to the path used above.